website/docs/plugins/submitit_launcher.md

```diff
@@ -73,7 +73,7 @@ params:
 
 You can set all these parameters in your configuration file and/or override them in the commandline: 
 ```text
-python foo.py hydra/launcher=submitit_slurm hydra.launcher.params.timeout_min=3
+python foo.py -m hydra/launcher=submitit_slurm hydra.launcher.params.timeout_min=3
```

Comment:
is it needed? I'm a bit confused

Reply:
Yes. see my comment on the original issue.